stacker_blueprints/conveyor/__init__.py

```diff
@@ -0,0 +1,482 @@
+from awacs.helpers.trust import get_default_assumerole_policy
+from stacker.blueprints.base import Blueprint
+from troposphere import (
+    autoscaling,
+    cloudformation as cf,
+    ec2,
+    elasticloadbalancing as elb,
+    logs,
+    sqs,
+    Base64,
+    Equals,
+    FindInMap,
+    GetAtt,
+    If,
+    Join,
+    Not,
+    Ref,
+)
+from troposphere.autoscaling import (
+    Metadata,
+    Tag as ASTag,
+)
+from troposphere.iam import (
+    InstanceProfile,
+    Policy,
+    Role,
+)
+from troposphere.policies import (
+    AutoScalingRollingUpdate,
+    AutoScalingScheduledAction,
+    UpdatePolicy,
+)
+from troposphere.route53 import RecordSetType
+
+from ..constants import ALLOWED_INSTANCE_TYPES
+from .policies import get_conveyor_policy
+
+CLUSTER_SG_NAME = 'ConveyorSecurityGroup'
+CONVEYOR_INSTANCE_PROFILE = 'ConveyorProfile'
+ELB_SG_NAME = 'ConveyorELBSecurityGroup'
+LAUNCH_CONFIGURATION = 'ConveyorLaunchConfiguration'
+LOAD_BALANCER = 'LoadBalancer'
+LOG_GROUP = 'LogGroup'
+QUEUE = 'Queue'
+
+
+class Conveyor(Blueprint):
+
+    LOCAL_PARAMETERS = {
+        'GitHubToken': {
+            'type': str,
+            'description': 'GitHub API token to use when creating commit statuses.',
+        },
+        'SshIdRsa': {
+            'type': str,
+            'description': 'SSH id_rsa for the bot github user.',
+        },
+        'SshIdRsaPub': {
```

Comment:
if you agree with above, `SshRsaPublicKey` - might want some comments that you actually need the contents of the files, not a filename.